Frontend/unhoused-outreach-ui/src/features/neighbors/housing-insecure/page-views/accordion.tsx

```diff
@@ -0,0 +1,76 @@
+import { useContext } from 'react';
+import { HousingInsecureNeighbor } from '../../../neighbors/housing-insecure';
+import { LookupsContext } from '../../../../App';
+import { Lookups } from '../../../lookups';
+import { Accordion, AccordionDetails, AccordionSummary } from '@mui/material';
+import { Link } from 'react-router-dom';
+import { ExpandMore } from '@mui/icons-material';
+import moment from 'moment';
+import { HousingInsecureNeighborPageProps } from '../../models/props';
+
+
+export function HousingInsecureNeighborsAccordionView({ housingInsecureNeighborsData }: HousingInsecureNeighborPageProps) {
+    // GET THE DATA.
+    const lookups = useContext(LookupsContext) as Lookups;
+
+    // FIELD FORMATTING FUNCTIONS.
+    const getFullName = (neighbor: HousingInsecureNeighbor): string => {
+        const firstName = neighbor.firstName ?? '';
+        const preferredName = neighbor.preferredName ? ` '${neighbor.preferredName}'`: '';
+        const lastName = neighbor.lastName ?? '';
+        return `${firstName}${preferredName} ${lastName}`
+    };
+    const getAge = (neighbor: HousingInsecureNeighbor): string => {
+        if (!neighbor.dateOfBirth) {
+            return 'Unknown';
+        }
+        const birthDate = moment(neighbor.dateOfBirth);
+        const currentDate = moment(new Date());
+        const age = currentDate.diff(birthDate, "years");
+        return String(age);
+    };
+    const getContact = (neighbor: HousingInsecureNeighbor): string => neighbor.phoneNumber ?? neighbor.emailAddress ?? 'N/A';
+    const getLocation = (neighbor: HousingInsecureNeighbor) => {
+        if (!neighbor.location) {
+            return null;
+        }
+        const mapLink = `map/${neighbor.location.latitude}/${neighbor.location.longitude}`;
+        return (
+            <>
+                Location:
+                <Link to={mapLink} style={{color: 'drakblue'}}>
+                    {neighbor.location.name ?? lookups.locationType[neighbor.location.locationTypeId]}
+                </Link>
+            </>
+        );
+    };
+
+    // CONTENT.
+    return (
+        <div style={{paddingTop: '20px'}}>
+        {housingInsecureNeighborsData.map(neighbor =>
+            <Accordion key={neighbor.housingInsecureNeighborId} className="accordion">
```

Comment:
just to make sure I'm reading this right - for the accordion we're using a material component but for the regular table we're doing reac-table?

Reply:
yeah. do you think we should use material elements for both as long as we're doing material for one of them?

Reply:
I'm not particularly familiar with react, but it seems a good idea to me to be consistent from a UI design perspective.

Reply:
The main reason I didn't think it was a huge deal is because we'll likely just use either the accordion or the table, not both. I'd like to mock up both to see which looks better. Although I'm almost entirely convinced right now it will be the accordion. And we might want to use material elements for other parts of the site, and if we did that, then the consistency throughout would be helpful. So I'm fine sticking more to material if that seems appropriate to everyone else.

Reply:
Yeah I really doubt we'll use a table, and if we do, let's use a material one, not react-table. I've removed everything related to the table and left the accordion. Let's use that unless we decide differently later.